BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BungeeListener.java

```diff
@@ -80,7 +84,7 @@ public void checkServerFull(ServerConnectEvent event) {
 		int current = MercuryAPI.getAllAccountsByServer(info.getName()).size() + QueueHandler.getPlayerOrder(info.getName()).size();
 		if (current >= count && !QueueHandler.isAllowedPassThrough(p.getUniqueId())) {
 			// Now we deal with redirecting the player.
-			db.setServer(p, info.getName());
+			p.setReconnectServer(info);
```

Comment:
I'm not 100% sure this calls the reconnect handler right away.  Does it do it in bungee?